src/proxy/router.js

```diff
@@ -31,8 +31,12 @@ export default class Router {
         else {
             var isStatic = typeof handler !== 'function';
 
-            if (isStatic)
+            if (isStatic) {
+                if (handler.isShadowUIStylesheet)
```

Comment:
I have feeling what it's not on it's place in route and it should be done in proxy. Maybe we can make method `_processStaticContent` which will do nothing in router, but will check flag and convert css in proxy